### DISCUSSION TITLE
SearchHandler BackButton does not trigger any event

### DISCUSSION THREAD

**Discussion by BadgeCom** [dotnet/maui#22074] [Q&A]
For the Shell Search Handler, when SearchBoxVisibility is set to Collapsible, there is a BackButton on the left side.
When I click the Back button it does not trigger any event or I do not know which event it may trigger. I have tested many events but haven't found one yet.

I need it because I want to hide my keyboard when closing the search handler with the Back button.

Shell.BackButtonBehaviour does not work for this.

Does anyone know which event I need to trigger or is there a workaround?

**Comment by zer0Ambit1on** [comment]
do you solve this?
